### ISSUE TITLE
Rows are in wrong order on right angle connectors

### ISSUE THREAD

**Issue by bobc** [KiCad/Connectors_IEC_DIN.pretty#5]
All male (right angle) connectors in this library are incorrect with respect to row order. Row a should be closest to connector body.

I don't have access to DIN spec, but sample vendor spec is
http://www.erni.com/fileadmin/medien/downloadcenter/din41612/ERNI-DIN41612-IEC60603-2-e.pdf

Fix: swap row a and row c.

**Comment by jkriege2** [comment]
The pin order was specifically designed this way. As you can see in the images below, they are made, so you can connect a pinheader to a socket-strip and get the same numbering!

![2017-01-29 14_11_17-3d viewer](https://cloud.githubusercontent.com/assets/11967523/23332621/140680f4-fb7d-11e6-8400-e78a5ee074d6.png)
![2017-01-29 14_08_02-pcbnew d__kicad_test_addlin_test_addlin kicad_pcb](https://cloud.githubusercontent.com/assets/11967523/23332622/14089330-fb7d-11e6-99fb-2f9be0a79e66.png)

**Comment by bobc** [comment]
Those are different connectors, completely irrelevant.

**Comment by jkriege2** [comment]
yep, mixed up the libs, that's why I also deleted the comment ...

BTW: Do you have the time to fix those footprints? Just go ahead!
Also: As you already noted these footprints all do not conform to KLC ... so you could fix them up in the process. 

Thinking about all this: We are currently cleaning up several libs, to get e.g. consistent naming schemes ... so I would seay we should move this whole lib into a new one `Connectors_IEC_DIN.pretty`
@SchrodingersGat What do you think? If OK, could you prepare that lib and declare this one deprecated?

cu
JAN

**Comment by SchrodingersGat** [comment]
We can simply rename this one and then any references to the old name are redirected. All that is then required is a change to the lib table

**Comment by jkriege2** [comment]
Wasn't there an issue that we cannot delete/rename a repo as then KiCAD with old projects that reference this repo crashes ... (heard somewhere ...)

cu
JAN

**Comment by SchrodingersGat** [comment]
Renaming has been performed multiple times ( e.g. ThroughHole -> THT

GitHub provides automatic redirection so it should be ok?

**Comment by jkriege2** [comment]
fine with me then!
JAN

**Comment by evanshultz** [comment]
@SchrodingersGat So will this library then get renamed? Are there any prior tasks to be done?

**Comment by SchrodingersGat** [comment]
@evanshultz this is one from the archives for sure! Had completely forgotten about this. I'm happy to rename this now. I'll also issue a PR updating the lib-table

**Comment by SchrodingersGat** [comment]
Ok this has been done - [lib-table PR here](https://github.com/KiCad/kicad-library/pull/1481)

**Comment by evanshultz** [comment]
@bobc Are you willing to fix these footprints? Filing an issue is nice, but providing a fix is far better.

**Comment by bobc** [comment]
I raised this issue because a user in the forum had a problem (toasters), I don't use them myself. [reference](https://forum.kicad.info/t/kicad-offical-din41612-connectors-mechanical-sockets-have-the-wrong-pin-order/5463)

I went looking for a script to generate them and then kinda forgot about it. Unfortunately, having to do extensive rework on symbols to comply with an ever changing KLC spec just to fix an underlying bug is quite a disincentive.

Personally, I would say if it is a "bugfix" change than that should have priority over KLC compliance.
